README.md

```diff
@@ -41,6 +41,16 @@ The following parameters are optional for your config file:
 All of the above attributes are required by the tap to connect to your mongo instance. 
 here is a [sample configuration file](./sample_config.json).
 
+Another option is to use a connection string. This method is most useful when
+connecting to Atlas. Please note this will ignore all parameters above. 
+
+
+```json
+{
+  "connection_string": "mongodb+srv://<username>:<password>@<hostname ip address>/<database name to sync from>?ssl=true"
```

Comment:
why not have a boolean flag, e.g `to_url`, that would instruct the tap to turn the same given config dictionary to a url string before creating the client. That way, it's compatible with the current config.

Reply:
Yep, that's a great idea! I didn't think about it as all I did was copy the URL from Atlas.
In fact, the difference here is in the `+srv` part, as SRV connections don't work when we pass a port.

Maybe the flag could be something like `use_srv`? Plus, what do you think if we add a `query_string` property, that would make the connection a bit more flexible?

Reply:
@Samira-El I just pushed a new version that should be more compatible with the current config. The solution I am proposing uses a connection string in any case but uses the same config.

I tested it manually with both SRV and standard connections.

Reply:
Thanks @ivanovyordan ! The PR looks better. I have left few comments